examples/complete/firestore/src/Todos.test.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { render, unmountComponentAtNode } from 'react-dom'
+import { act } from 'react-dom/test-utils'
+import { useSelector } from 'react-redux'
+import Todos from './Todos'
+
+jest.mock('react-redux')
+
+describe('Should test the sample component', () => {
+  let container = null
+  const expectedTodos = []
+
+  beforeEach(() => {
+    // setup a DOM element as a render target
+    container = document.createElement('div')
+    document.body.appendChild(container)
+  })
+
+  afterEach(() => {
+    // cleanup on exiting
+    unmountComponentAtNode(container)
+    container.remove()
+    container = null
+  })
+
+  it('show empty text when there is no text', () => {
+    useSelector.mockReturnValue(expectedTodos)
```

Comment:
Can we make `expectedTodos` within the scope of the test instead of global to all tests?

Reply:
I will change it.